src/pronoun/declensionPronoun.ts

```diff
@@ -298,3 +280,39 @@ export function declensionPronoun(
   }
   return null;
 }
+
+const DEMONSTRATIVE_PRONOUNS: Record<string, string> = {
+  ona: 'onoj',
+  ono: 'onoj',
+  ota: 'otoj',
+  oto: 'otoj',
+  ova: 'ov',
+  ovo: 'ov',
+  tamta: 'tamtoj',
+  tamto: 'tamtoj',
+  tuta: 'tutoj',
+  tuto: 'tutoj',
+  ta: 'toj',
+  to: 'toj',
+  te: 'toj',
+  vsi: 'veś',
+  vse: 'veś',
+};
+
+const DEMONSTRATIVE_POSTFIXES: Record<string, string> = {
+  ze: 'že',
+  koli: 'koli',
+  libo: '-libo',
+  nebud: '-nebųď',
+};
+
+function splitPostfix(original: string, standard: string) {
```

Comment:
Refactored if-else to regular expressions and a small lookup map. All tests pass.